terraform/sql-database-server/main.tf

```diff
@@ -1,6 +1,6 @@
 # https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/sql_server
 
-resource "azurerm_sql_server" "adl_sql" {
+resource "azurerm_mssql_server" "adl_sql" {
```

Comment:
⚠️ **[tfsec]** <[AVD-AZU-0027](https://aquasecurity.github.io/tfsec/v1.28.1/checks/azure/database/enable-audit/)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Server does not have an extended audit policy configured.